src/style.css

```
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(-n+1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
```

```suggestion
.stars--1 :nth-child(-n+1),
.stars--2 :nth-child(-n+2) {
  background-image: url(./images/star-active.svg);
}
```
list all selectors to apply the same style for each one